infrastructures/kubernetes/src/main/java/org/eclipse/che/workspace/infrastructure/kubernetes/namespace/pvc/UniqueWorkspacePVCStrategy.java

```diff
@@ -174,7 +174,7 @@ else if (provisionedVolumeName2PVC.containsKey(volumeName)) {
   @Override
   public void cleanup(String workspaceId) throws InfrastructureException {
     clientFactory
-        .create()
+        .create(workspaceId)
         .persistentVolumeClaims()
         .inNamespace(namespaceName)
```

Comment:
@garagatyi I'm not sure that we can do it while configuring a client. Now we do such configuring by using `KubernetesNamespace`/`OpenShiftProject` clients wrappers. Maybe we should use `KubernetesPersistentVolumeClaims` from `KubernetesNamespace` here instead of directly using of kubernetes client.